pkg/tests/apis/playlist/playlist_test.go

```diff
@@ -169,6 +171,33 @@ func TestIntegrationPlaylist(t *testing.T) {
 				featuremgmt.FlagKubernetesPlaylists, // Required so that legacy calls are also written
 			},
 		}))
+
+		client := helper.GetResourceClient(apis.ResourceClientArgs{
+			User: helper.Org1.Editor,
+			GVR:  gvr,
+		})
+
+		// Folder support needs to be enabled explicitly for this resource
```

Comment:
Adding an integration test on playlist (only mode5) that ensures we do not write values with folder support enabled